internal/format/testdata/asciidoc/document-NoHeader.golden

```diff
@@ -1,3 +1,10 @@
+== Resources
```

Comment:
All the `*.golden` files look great, unless you agreed to my comments about some reordering and empty lines! :+1:

Reply:
I made the suggested changes and updated the `*.golden` files accordingly.